### ISSUE TITLE
Remove spam log please

### ISSUE THREAD

**Issue by exodius48** [liudongmiao/ForceStopGB#15]
Prevent running always spam xposed log with [V/Prevent] .... or [D/Prevent] ...

**Comment by liudongmiao** [comment]
Actually, it's very useful for debug...

I add the [V/Prevent] and [D/Prevent] for better debug.

**Comment by DeadneM** [comment]
An option to disable verbose mode would be appreciated 
Because for example when i provide log for other module there is lot i mean hundred hundred line with v/prevent or d/prevent

**Comment by liudongmiao** [comment]
1. it's very easy to filter it out.
2. it's very hard to find the problem if there is no log.
3. it's part of xposed module way for prevent running. if you don't want to it as xposed module, you can use it without xposed installer. (However, I only write a chinese document.)

**Comment by exodius48** [comment]
Hmm.. but it make my xposed log bigger & bigger over time. Also it will make hard to find error log from anoter module/s.

**Comment by liudongmiao** [comment]
Currently, when prevent running is activated, it will only record other level log, like "[I/Prevent]", "[W/Prevent]", and "[E/Prevent]".

The "[V/Prevent]" and "[D/Prevent]" only exist when it's not activated.

If there are many "V" and "D" log, please report bug click the menu. And if you think some "I" message should be "D", please report bug too. 
(report bug means send some xposed log, you can use log in xposed, or just click "report bug" in prevent running).

**Comment by DeadneM** [comment]
Yes but i think what was wanted is to disable log from your module with an on/off button in the application settings, some module can do that amplify for exemple 

It is a feature request not a problem with log or app by itself 

Happy Valentine day

**Comment by exodius48** [comment]
@liudongmiao i already pm you on xda about error log, please check bro.
